monitoring/monitoring-client.py

```diff
@@ -20,16 +20,16 @@ def alert(data):
         info["embeds"].append({'description': data})
         try:
             r = requests.post(url, data=json.dumps(info), headers={'Content-type': 'application/json'})
-        except Exception as e:
+            print_to_file("Alerting (discord) - {}: {}\n".format(r.status_code, r.reason))
```

Comment:
Put this line in an 'else:' block following the 'except:' block. The else block will run if no exception is caught and will also keep the try/except from throwing an exception if the file write fails in print_to_file().

For reference: https://docs.python.org/3/tutorial/errors.html#handling-exceptions
Ctrl+F for 'else clause'